java-util/src/main/java/io/druid/java/util/common/granularity/Granularity.java

```diff
@@ -150,6 +148,10 @@ public int compare(Granularity g1, Granularity g2)
 
   public abstract DateTime toDate(String filePath, Formatter formatter);
 
+  public DateTime nextRound(DateTime time) {
```

Comment:
tbh - i think this is cosmetic and can be dealt with later even after the merge. The method names from refactor preserve the same names as today. so as long as we are not changing existing ones, it works for me.